Makefile

```diff
@@ -63,35 +63,28 @@ install: # Install setuptools, install this package in editable mode
 format: setup.py pyproject.toml $(MAKE_SOURCES) # Run the code formatter and import sorter
 	-black $(LOCATIONS)
 	-isort $(LOCATIONS)
-	@echo "Ignore, Created by Makefile, `date`" > $@
```

Comment:
I'm tired of having these things floating around and need to either gitignore or remove. I opted for the latter. (I simply don't think we use them for anything)